dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -116,3 +116,22 @@ object reflects the newly requested size in `PersistentVolume.Spec.Capacity`. At
 this point, you can create or re-create a new pod from the PVC to finish the
 file system resizing. Once the pod is running, the newly requested size is
 available and `FileSystemResizePending` condition is removed from the PVC.
+
+[[recover_from_resize_failure]]
+== Recovering from Failure when Expanding Volumes
+
+If expanding underlying storage fails either on master or node,
+{product-title} admin can manually recover the PVC state and cancel the resize requests that are
+continuously retried by the controller without administrator intervention.
+
+Currently, this can be done manually by completing the following steps:
+
+1. Mark the PV that is bound to the claim (PVC) with the `Retain` reclaim policy.
+   This can be done by editing the PV and changing persistentVolumeReclaimPolicy to `Retain`.
+2. Delete the PVC, so as we can re-create it later.
+3. To ensure that newly created PVC can bind to PV marked `Retain`, we will have to manually edit the PV
```

Comment:
Change to "To ensure that the newly created PVC can bind to the PV marked `Retain`, manually edit the PV and delete the `claimRef` entry from the PV specs. This marks the PV as `Available.` 

Change to "For more information about prebinding PVCs, see xref:./persistent_volumes.adoc#persistent-volumes-volumes-and-claim-prebinding[volume and claim prebinding]."